### ISSUE TITLE
Add Original game rating to game page/homepage (where ever theres games)

### ISSUE THREAD

**Issue by sha4owz** [tersiswilvin/Roblox-2019-Old-Theme#12]
![yc3b2vfhr3cb1](https://github.com/tersiswilvin/Roblox-2019-Old-Theme/assets/138281770/2b4f2916-e209-4058-899b-d1351eaa1739)

**Comment by sha4owz** [comment]
Its the little thumbs up with the bars/ratings next it under each game

**Comment by tersiswilvin** [comment]
Hi, I had considered adding this in the past when I was adding 2019E and pre-2018 however I realized it would not work with CSS alone which was why I didn't add it in when adding those options, the same is said about the creator name when hovering over the option as there is no way to get the creator name of each game with just CSS unless it was in the game card already.

**Comment by ghost** [comment]
Would it be possible to use JavaScript? #57

**Comment by tersiswilvin** [comment]
> Would it be possible to use JavaScript?

Yes it is possible to use javascript, however it will most likely won't be planned for awhile.
